tests/bud.bats

```diff
@@ -390,6 +390,25 @@ _EOF
   assert "$output" !~ "unwanted stage"
 }
 
+# Note: Please skip this tests in case of podman-remote build
+@test "build: test race in updating image name while performing parallel commits" {
+  # Run 25 parallel builds using the same Containerfile
+  local count=25
+  for i in $(seq --format '%02g' 1 $count); do
+      timeout --foreground -v --kill=10 60 \
+              ${BUILDAH_BINARY} ${BUILDAH_REGISTRY_OPTS} ${ROOTDIR_OPTS} $WITH_POLICY_JSON build --squash --iidfile ${TEST_SCRATCH_DIR}/id.$i --timestamp 0 -f $BUDFILES/check-race/Containerfile &>/dev/null &
+  done
+  # Wait for all background builds to complete. Note that this succeeds
+  # even if some of the individual builds fail! Our actual test is below.
+  wait
+  # Number of output bytes must be always same, which confirms that there is no race.
+  for i in $(seq --format '%02g' 1 $count); do
+      test $(cat ${TEST_SCRATCH_DIR}/id.$i|wc -c) = 71
```

Comment:
Please do not use `test`. Do not ever use `test`. That gives no information whatsoever to the poor person trying to debug a test failure. Please use `assert`.

Reply:
So, going back to the first question, you can rewrite it the first way as
```bash
    assert "$(cat ${TEST_SCRATCH_DIR}/id.* | wc -c)" = 1775 "Total chars in all id.* files"
```

Reply:
Thanks @edsantiago , Amended, I'll wait and hope CI goes green now ( passes on my local )